models/User.model.js

```diff
@@ -3,10 +3,13 @@ const Schema = mongoose.Schema;
 
 const userSchema = Schema(
   {
+    userId: Number,
```

Comment:
We wouldn't need to add a `userId` to the `User` model because Mongoose will figure out how to do this on it's own.